0800-appendix-195943-stock.Rmd

```diff
@@ -16,38 +16,28 @@ source('scripts/packages.R')
 ```{r  load-data-195943}
 my_site <- 195943
 my_site2 <- 195944
-
-
-
-
+my_site3 <- 1805507
 ```
 
 `r fpr::fpr_appendix_title()`
 
 ## Site Location {-}
 
 PSCIS crossing `r as.character(my_site)` is located on the Upper Bulkley
```

Comment:
Stock Creek is not on the Upper Bulkley.  Upper Bulkley is above the confluence with the Morice